x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -643,6 +760,34 @@ export const ModelsList: FC<Props> = ({
           onTableChange={onTableChange}
           sorting={sorting}
           data-test-subj={isLoading ? 'mlModelsTable loading' : 'mlModelsTable loaded'}
+          childrenBetween={
+            isCalloutVisible ? (
+              <>
+                <EuiCallOut
+                  size="s"
+                  title={
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.modelsList.newElserModelTitle"
+                      defaultMessage="New ELSER model now available"
+                    />
+                  }
+                  onDismiss={setIsCalloutVisible.bind(null, false)}
+                >
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.modelsList.newElserModelDescription"
+                    defaultMessage="ELSER model v2 release shows faster performance and better relevance for improved information retrieval."
```

Comment:
```suggestion
                    defaultMessage="The ELSER v2 model has improved speed and relevance, delivering more performant information retrieval."
```

Totally optional rewording suggestion :)

Reply:
Do we want to substitue "information retrieval" for "text expansion" or "semantic search"?